demo/src/screens/MainScreen.js

```diff
@@ -241,11 +242,9 @@ class MainScreen extends Component {
         floatingPlaceholder={false}
         text70
         leadingAccessory={
-          !filterText ? (
-            <View>
-              <Icon tintColor={Colors.$iconDefault} source={Assets.icons.demo.search}/>
-            </View>
-          ) : undefined
+          <View>
```

Comment:
I think you can remove the wrapping View.
Also, let's add a small margin (s2) to the icon on the right to keep it a little spaced from the text

Reply:
About the margin you meant for the left button the search button right?

Reply:
Yes, the left icon (the one I comment about)

Reply:
done

Reply:
I think you missed this (: ?
> I think you can remove the wrapping View.